### PR TITLE
Reduce minimum event counts and number of time cut-offs

### PR DESCRIPTION
**Pull request by kurttaylor**
The aim of this branch is twofold:

1. Reduce the minimum number of events required for the models to run (currently 400, reducing down to 50) as hospitalized phenotypes were not running.
2. Reduce the number of time cut-offs in the models - I am not sure it is possible for the models to run the reduced cut-offs _**as well as**_ the normal cut-offs so I have just changed the normal ones for now to get results with the reduced cut-offs. We will have to change this back if we want to go back to the previous cut-offs. Happy to change this if there is a different way?

### CODE REVIEW

File: analysis/02_pipe.R [hunk 1]
```diff
@@ -44,7 +44,7 @@ cohort_end_date <- as.Date("2021-12-14")
 #Used to split time since COVID exposure; when there are time periods with no events then
 #a reduced number of time periods is used (need 197 instead of 196 as time periods are split using [ , ) 
 
-cuts_days_since_expo <- c(14, 28, 56, 84, 197) 
+cuts_days_since_expo <- c(28, 197) 
 cuts_days_since_expo_reduced <- c(28,197)
```
Comment (GenevieveCezard) [inline analysis/02_pipe.R:48]: If we change `cuts_days_since_expo` to `c(28, 197)` then `cuts_days_since_expo_reduced` becomes redundant. I think we should keep the 2 sets of time cuts and make sure the reduced version is used any when we have a reduced number of events.

**Review by GenevieveCezard — COMMENTED** [review]
Changes seem fine overall. 
I just wonder whether we should keep the set with a higher number of time cuts as it will look better in figures for outcomes with a high number of events.

File: analysis/02_pipe.R [hunk 2]
```diff
@@ -44,7 +44,7 @@ cohort_end_date <- as.Date("2021-12-14")
 #Used to split time since COVID exposure; when there are time periods with no events then
 #a reduced number of time periods is used (need 197 instead of 196 as time periods are split using [ , ) 
 
-cuts_days_since_expo <- c(14, 28, 56, 84, 197)
```
[the remark this answers is not in the document]
Comment (kurttaylor) [reply analysis/02_pipe.R:47]: I have made the requested change. I agree that this is something that should be an extra parameter. I have created an issue to reflect this ([here](https://github.com/opensafely/post-covid-vaccinated/issues/151)).

[the remark this answers is not in the document]
Comment (GenevieveCezard) [reply analysis/02_pipe.R:47]: Thank you for creating an issue for this

**Review by GenevieveCezard — APPROVED** [review]

**Pull request merged**